### PR TITLE
Add plugin to testgrid dashboard

### PR DESCRIPTION
**Pull request by brybacki**
Signed-off-by: Bartosz Rybacki <brybacki@redhat.com>

### CODE REVIEW

**Comment by dhiller** [comment]
Hm, strange, when I run the check w/ phaino it complains about some config error. @brianmcarey do you know whether we still suffer from the greenhouse cache? I suspect this is a missing cache entry somehow?

Side node: when running on the default branch the check succeeds...

**Comment by brianmcarey** [comment]
@dhiller the greenhouse cache was cleared on the main workloads cluster - I can try and clear the cache on this cluster too to see if it resolves some these issues?

**Comment by brybacki** [comment]
@brianmcarey did you clear any other cache?
anyway, i can try again
/test pull-project-infra-check-testgrid-config

**Comment by brybacki** [comment]
I do not understand that error. Does it mean my change is missing something?

**Comment by brybacki** [comment]
@dhiller @brianmcarey I do not understand what is wrong here, what can I do to troubleshoot what is missing?

**Comment by brianmcarey** [comment]
@brybacki pull-project-infra-check-testgrid-config  is failing across all PRs and is set to be not required. I am looking into getting this job passing again - this is not related to your changes.

**Comment by dhiller** [comment]
@brybacki there's a fix for the check in place that needs to get in first: https://github.com/kubevirt/project-infra/pull/2044

**Comment by brybacki** [comment]
@dhiller great, it is finally merged, do I need to rebase?

File: github/ci/testgrid/gen-config.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ dashboards:
 - name: kubevirt-hyperconverged-cluster-operator-presubmits
 - name: kubevirt-hyperconverged-cluster-operator-periodics
 - name: kubevirt-hyperconverged-cluster-operator-openshift-ci-presubmits
+- name: kubevirt-velero-plugin-presubmits
   dashboard_tab:
```
[the remark this answers is not in the document]
Comment (brybacki) [reply github/ci/testgrid/gen-config.yaml:10]: yes, I am glad that the script and tests finally work correctly, so I can fix the problem

[the remark this answers is not in the document]
Comment (brybacki) [reply github/ci/testgrid/gen-config.yaml]: corrected

**Review by brianmcarey — COMMENTED** [review]
/lgtm

**Review by dhiller — COMMENTED** [review]
/approve

Apologies for the long delay.

**Pull request merged**